azure-pipelines.yml

```diff
@@ -168,8 +181,39 @@ stages:
             bash <(curl -s https://codecov.io/bash)
           displayName: 'Code coverage'
 
+  # C) Create a relase
+  # Activated when all these are met:
+  # - is a manual trigger
+  # - is on branch master
+  # - DO_RELEASE is true
+  - stage: Release
+    condition: 
+      and(
+        succeeded(),
+        and (
+          eq(variables['DO_RELEASE'], true),
+          and(
+            eq(variables['Build.SourceBranch'], 'refs/heads/master'),
+            eq(variables['Build.Reason'], 'Manual')
+          )
+        )
+      )  
+    dependsOn:
+      - Build
+      - Test
+    pool:
+      vmImage: 'ubuntu-latest'
+    jobs:
+      - job: make_release      
+        steps:
+        - template: azure-templates/make-release-steps.yml
+          parameters:
+            semver: '${{ parameters.RELEASE_SEMVER }}'
+            gitEmail: $(GIT_EMAIL)
+            gitUsername: $(GIT_USERNAME)
+            gitHubConnection: $(GITHUB_CONNECTION)
```

Comment:
can we set these ones at organization level ? to avoid setting these for every pipeline / project

Reply:
I do not think it's possible. I know there is a way to share variables across multiple pipelines using variable group but the pipelines must be located in the same project (i.e. not at organization level). 

By the way, maybe it could be possible to use a custom solution to get this same result (e.g. just reading a well-known public file available on GitHub, e.g. https://github.com/pagopa/io-pagopa-commons/xxx)...

Reply:
if they're fixed we can consider hardcoding them into the .yml (and distribute the files using code generation)

Reply:
I think they're just fine as variables as we may need to re-configure them and that should not impact source code.

Moreover, connection is an information that really has meaning only inside Azure DevOps settings and email/username are loosely bound to that.

Given that pipelines have to be configured anyway, I'd not over-engineer on this and just use variables.

Reply:
Maybe we can add a default hardcoded value here and pass an override parameter in azure-pipelines.yml only when the value is different.

https://github.com/pagopa/io-functions-admin/blob/175476527-release-pipeline/azure-templates/make-release-steps.yml#L12

Reply:
It seems not possibile set a variable on organization level.